.github/workflows/flask-base.yml

```diff
@@ -33,9 +33,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
```

Comment:
Won't this run with python2?

Reply:
AFAIK the setup-python action above, makes sure I have the correct python version.